third_party/amd/lib/TritonAMDGPUTransforms/BlockPingpong.cpp

```diff
@@ -296,7 +314,11 @@ LogicalResult Pingponger::transformFourPPClusters(OpBuilder &builder,
   appendClusterBarrier(builder, loc);
 
   // mem3: local store A and B
-  updateOpInsertion(lStoreOps[1]);
+  // Persistent matmul kernels may include an epilogue as part of the main
```

Comment:
Agreed, it's generally true for any matmul kernel.

Reply:
Thanks. I'll update this comment to generalize.